src/main/java/mate/academy/service/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.service;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://127.0.0.1:3306/test";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user","root");
+        DB_PROPERTIES.put("password", "Nero0410");
```

Comment:
Storing credentials in the code is a bad practice due to security reasons. It's better to use environment variables or a configuration file that is not included in your version control.